server/src/main/java/org/opensearch/index/search/stats/SearchStats.java

```diff
@@ -239,6 +244,8 @@ private Stats(StreamInput in) throws IOException {
             suggestTimeInMillis = in.readVLong();
             suggestCurrent = in.readVLong();
 
+            searchIdleWakenUpCount = in.readVLong();
```

Comment:
Since you are serializing this field you will have to gate it with version checks like so:
```
if (in.getVersion().onOrAfter(Version.V_3_0_0)) {
    ...
}
```
 Also, we typically put new fields at the end of the stream.